### ISSUE TITLE
Satellite UI: add bucket settings menu to bucket file view

### ISSUE THREAD

**Issue by mobyvb** [storj/storj#6811]
Add a menu when the bucket is opened which allows you to do the following:
* toggle Media Type (MIME Type) vs. extension type for the "Type" column in table
* share bucket
* delete bucket
* Enable/Suspend versioning

More info on media types and available formats:
https://www.iana.org/assignments/media-types/media-types.xhtml
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types

Screenshot (Show Media Type):
<img width="558" alt="media" src="https://github.com/storj/storj/assets/3217669/b468aa8e-efe1-4c6d-ba45-0830ebdfa15a">

Screenshot (Show File Type):
<img width="567" alt="type" src="https://github.com/storj/storj/assets/3217669/6e4da9ff-3fb4-467f-8e60-2fc2898c6919">

### AC:
* add a page-level settings menu on the bucket file view
    * toggle Show Media Type (MIME) vs. Show File Type (inferring type based on file extension)
    * share bucket
    * delete bucket (redirects to bucket list page after delete)
    * Enable/Suspend versioning

**Comment by wilfred-asomanii** [comment]
On investigating, I realised displaying the media type of uploaded objects will not be as simple as we originally thought.
My main source is from [this answer](https://stackoverflow.com/a/51994838) in stack overflow, but I have tested the `GetObject.ContentType` to verify this.
Summarising the answer, uploading programatically as we do and do not provide a `ContentType`, the default type assigned to the object is `application/octect-stream` (or `binary/octect-stream` in the answer).
This means we will have to use other means to get infer the content type from the object data itself.
